### DISCUSSION TITLE
Will be a self-hosted photoprism exposed to the Internet indexed by search engines?

### DISCUSSION THREAD

**Discussion by racecrew** [photoprism/photoprism#3691] [Q&A]
The indexing of the app can be restricted a file like robots.txt
Is this by default there in the app? Or should it be manually mounted? If it should be mounted, would you pls provide a short example how to do it exactly? 

Thanks!

**Comment by lastzero** [comment]
I guess we'd need yet another config option for that. Some people may want their instance to be indexed.

**Reply by racecrew** [reply]
Ok, then my question would be a feature request ;-)
